index.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+module.exports = () => ({
+    transform: {
+        '\\.js$': require.resolve('babel-jest'),
+        '\\.(png|jpg|jpeg|gif|webp|ico)$': require.resolve('jest-file'),
+        '\\.(eot|ttf|woff|woff2|otf)$': require.resolve('jest-file'),
+        '\\.(mp3|flac|wav|aac|ogg|oga|mp4|m4a|webm|ogv)$': require.resolve('jest-file'),
+        '\\.(svg)$': require.resolve('jest-file'),
```

Comment:
I think we are missing the `data-url` extensions and `inline` as they should have different return values other than a file path.

I don't know if using what currently is yields problems.. I'm more worried about the `inline` as it will be used as `dangerouslySetInnerHTML` in most cases and might cause react to crash.

Something to investigate.

Reply:
This is indeed an issue and recreating the behaviors we're using in our boilerplate requires using tailored transformers to replicate them.

Though i'm not sure if we should fix that here, if this is supposed to be a general use config. I think that should be tackled on the boilerplate itself.